cdap-common/src/main/java/co/cask/cdap/common/utils/FileUtils.java

```diff
@@ -35,4 +35,72 @@
 
   private FileUtils(){ }
 
+  /**
+   * Convert a permission, interpreted as a 3-digit octal, into a umask that yields this permission.
```

Comment:
It is not an acronym. It is commonly known as a "umask" in Unix and Hadoop jargon.